charts/tornjak/templates/server-statefulset.tpl

```diff
@@ -123,7 +119,9 @@ spec:
         - name: spire-config
           configMap:
             name: spire-server
+        - name: spire-server-socket
 {{- if not .Values.OIDC.enable }}
+          emptyDir: {}
```

Comment:
This is functionally correct but I think its a bit messy, I think it would help redability if we had the if else clause just for `spire-server-socket` volume, and then have another if else clause for the rest of the volumes. OR just put the `-name: spire-server-socket` line inside the if else black.